README.md

```diff
@@ -66,24 +67,25 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath group: 'com.anaplan.engineering', name: 'vdm-gradle-plugin', version: '2.6.4'
+    classpath group: 'com.anaplan.engineering', name: 'vdm-gradle-plugin', version: '3.0.2'
   }
 }
  
 apply plugin: 'vdm'
 ```
+The plugin will automatically pick up any annotation classes that have been added to the classpath dependencies.
 
-The plugins tasks are then invoked as they would be for any other Gradle build, for example:
+The tasks of the plugin are then invoked as they would be for any other Gradle build, for example:
 ```bash
 gradle clean build
 ``` 
 
-The tasks that the plugin makes available are describe in [tasks](#tasks) below.
+The tasks that the plugin makes available are described in [tasks](#tasks) below.
 
 The behaviour of the plugin can be customized by adding additional configuration to the `build.gradle` file. The options available are described in [configuration](#configuration) below. 
 
 ### Publishing artifacts
-It is trivial to instruct Gradle to publish the specification to a Maven-formatted artifact repository using the maven-publish plugin. The vdm-gradle-plugin will automatically mark artifacts`—according to the [configuration](#configuration)`—for publish when the maven-publish plugin has been applied to the project. 
+It is trivial to instruct Gradle to publish the specification to a Maven-formatted artifact repository using the maven-publish plugin. The vdm-gradle-plugin will automatically mark artifacts — according to the [configuration](#configuration)— for publishing when the maven-publish plugin has been applied to the project. 
```

Comment:
I tend to favor American spelling/punctuation (em dash, no spaces) etc in these docs in accordance with our style guide. Although it's fair to say I might not be completely consistent :)